src/app/contact/page.tsx

```diff
@@ -0,0 +1,177 @@
+'use client';
+
+import React, { useRef } from 'react';
+import { motion } from 'framer-motion';
+import { BsArrowRight } from 'react-icons/bs';
+import {
+  HiOutlineMapPin,
+  HiOutlineEnvelope,
+  HiOutlinePhone,
+} from 'react-icons/hi2';
+import { ToastContainer, toast } from 'react-toastify';
+import { fadeIn } from '../../utils/variants';
+import emailjs from '@emailjs/browser';
+import 'react-toastify/dist/ReactToastify.css';
+import Map from '../../components/map';
+
+export default function ContactPage() {
+  const ref: any = useRef();
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+
+    const formData = new FormData(ref.current);
+    const requiredFields = ['user_name', 'user_email', 'subject', 'message'];
+    let isFormValid = true;
+
+    requiredFields.forEach((field) => {
+      const value = formData.get(field);
+      if (!value || '') {
+        toast.error(`Please fill in ${field} field.`, {
+          theme: 'dark',
+        });
+        isFormValid = false;
+      }
+    });
+
+    if (!isFormValid) {
+      return;
+    }
+
+    emailjs
+      .sendForm(
+        'service_ap4wbtw',
+        'template_grf4xdq',
+        ref.current,
+        'dStWJXq6rdaw4xHWs'
+      )
+      .then(
+        () => {
+          toast.success('Message sent successfully!', {
+            theme: 'dark',
+          });
+
+          ref.current.reset();
+        },
+        () => {
+          toast.error('Something went wrong!', {
+            theme: 'dark',
+          });
+        }
+      );
+  };
+
+  return (
+    <>
+      <div className='w-full h-full px-6 md:px-0'>
+        <div className='container flex items-center justify-center h-full mx-auto text-center md:py-4 xl:text-left'>
+          <div className='flex flex-col w-full max-w-[700px]'>
+            <motion.h2
+              variants={fadeIn('up', 0.2)}
+              initial='hidden'
+              animate='show'
+              exit='hidden'
+              className='mb-12 text-center text-[32px] md:text-[48px] leading-tight font-semibold'
+            >
+              Let's <span className='text-[#1E50FF]'>connect.</span>
```

Comment:
76:18  Error: `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.  react/no-unescaped-entities